changelog.txt

```diff
@@ -1,3 +1,15 @@
+4.26.0
+==========
+### New Features: 
+**MSAL.NET now has support for MSA-passthrough with WAM** See Issue [#2126](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2126)
+**MSAL.NET now logs telemetry for the cache refresh status** See Issue [#2356](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2356)
+**MSAL.NET will now specify the tenant id for application token cache keys** See Issue[#2381](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2381)
```

Comment:
```suggestion
**MSAL.NET now uses ClientID+tenantID instead of just ClientId in the computation of the `SuggestedCacheKey` for `AcquireTokenForClient` (client credentials). This helps keeping the cache smaller in multi-tenant confidential client applications.  ** See Issue[#2381](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2381)
```